governance/polls/templates/Generic/$GenericApprovalWithMajority - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,66 @@
+---
+title: $GenericChange - $date_MONTH_DD,_YYYY
+summary: Signal your support or opposition to $GenericChange.
+discussion_link: $discussion_link
+parameters:
+  input_format: 
+    type: choose-free
+    options: [1,2,3]
+    abstain: [0]
+  victory_conditions:
+    - { 
+        type: 'and', 
+        conditions: [
+          { type : approval },
+          { type: majority, percent: 50 },
+        ]
+      }
```

Comment:
Can rewrite as if you want:
  ```
victory_conditions:
    - type: and
      conditions:
          - { type : approval }
          - { type: majority, percent: 50 }
    - { type : default, value : $StatusQuoOrRejectOptionNumber }
```

Reply:
Slightly cleaner.

Reply:
Will this definitely work? You have a greater grasp of yaml than I do. I know that the way I've written it worked in the Goerli polls we did